test/jobs/monitor_job_test.rb

```diff
@@ -0,0 +1,117 @@
+require 'quick_test_helper'
+require 'minitest/autorun'
+require 'mocha/setup'
+rails_require 'jobs/monitor_job'
+
+unless defined? RESQUES
+  RESQUES = []
+end
+unless defined? Rails
+  Rails = Module.new
+end
+
+class MonitorJobTest < MiniTest::Test
+  include Mocha::API
```

Comment:
For whatever reason, the Mocha setup instructions differ from the code and neither works enough to set up Mocha in the tests properly, so it seems I have to do all this manually.  Which is fine—I'd like to keep mocking at a minimum